oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -120,9 +120,6 @@ node.".format(e))
                     if storage.lower() == 's3' and bucket.lower() != 'some_bucket':
                         dbrmroot = BYPASS_SM_PATH
 
-                        if not os.path.exists(dbrmroot):
-                            os.makedirs(dbrmroot)
```

Comment:
Creating a directory, which is creating a loadbrm failure when attempted to load this directory as brm

Reply:
Does it mean that we have to create directories beforehand ?